responsive slider

```diff
@@ -0,0 +1,160 @@
+    </head>
+    <!--body starts here -->
+
+    <body>
+        <div class="container">
+            <div class="content">
+                <div>
+                    <ul class="nu">
+                        <li>100 -</li>
+                        <li>90 -</li>
+                        <li>80 -</li>
+                        <li>70 -</li>
+                        <li>60 -</li>
+                        <li>50 -</li>
+                        <li>40 -</li>
+                        <li>30 -</li>
+                        <li>20 -</li>
+                        <li>10 -</li>
+                        <li> 0 - </li>
+                    </ul>
+                </div>
+                <div class="align">
+                    <div class="range">
+                        <div id="slider">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <script>
+
+                const slider = document.querySelector(`#slider`);
+                const range = document.querySelector(`.range`);
+                const nu = document.querySelector(`.nu`)
+
+                /*
+    
+                arrow function below consist of if statements checks that check for parameters 
+                (buttonSize, colorTheme, sliderSize, buttonPosition) 
+                buttonSize = [small,medium,large]
+                colorTheme = [blue, red]
+                sliderSize = adjusts size of button [small,medium,large]
+                buttonPosition = postion of the button on track bar (0px-160px) 
+                */
+
+                const sliderFunc = (buttonSize, colorTheme, sliderSize, buttonPosition) => {
+                    if (buttonSize === 'small') {
+                        slider.style.height = '20px';
+                        slider.style.width = '10px';
+                    } else if (buttonSize === 'medium') {
+                        slider.style.height = '30px';
+                        slider.style.width = '15px';
+                    } else {
+                        slider.style.height = '40px';
+                        slider.style.width = '20px';
+                    }
+
+                    if (colorTheme === 'blue') {
+                        range.style.backgroundColor = 'blue';
+                        slider.style.backgroundColor = 'yellow';
+                        nu.style.color = 'blue';
+                    } else if (colorTheme === 'red') {
+                        range.style.backgroundColor = 'red';
+                        slider.style.backgroundColor = 'orange';
+                        nu.style.color = 'red';
+                    }
+
+                    if (sliderSize === 'small') {
+                        range.style.height = '200px';
+                    } else if (buttonSize === 'medium') {
+                        range.style.height = '300px';
+                    } else {
+                        range.style.height = '400px';
+                    }
```

Comment:
Please investigate this part of the function. Right now it is only partially working.